tests/core/resource/test_viewset.py

```diff
@@ -119,12 +114,12 @@ def test_class_parameters_are_used_for_collection_arguments(self, mocked):
             collection_get_arguments=collection_get_arguments
         )
 
-        arguments = viewset.collection_arguments(mock.MagicMock, 'get')
+        arguments = viewset.collection_arguments(mock.MagicMock(), 'get')
 
         self.assertDictEqual(
             arguments,
             {
-                'schema': mocked(),
+                'schema': mocked().bind(),
```

Comment:
This is the return value if we call `bind()` on a mocked schema. If we want to avoid this, we should also mock patch the bind value. egs.
```python
>>> m = mock.MagicMock()
>>> m.method()
<MagicMock name='mock.method()' id='4417348560'>
```
And a proposed solution:
```
>>> m = mock.MagicMock()
>>> m.method = lambda: m
>>> m.method()
<MagicMock id='4417096080'>
```